caddyhttp/fastcgi/fastcgi.go

```diff
@@ -248,6 +249,11 @@ func (h Handler) buildEnv(r *http.Request, rule Rule, fpath string) (map[string]
 	// Strip PATH_INFO from SCRIPT_NAME
 	scriptName = strings.TrimSuffix(scriptName, pathInfo)
 
+	// Add vhost path prefix to scriptName. Otherwise, some PHP software will
+	// have difficulty discovering its URL.
+	pathPrefix, _ := r.Context().Value(caddy.CtxKey("path_prefix")).(string)
+	scriptName = pathPrefix + scriptName
```

Comment:
Maybe `path.Join` here for consistency in case pathPrefix has a trailing slash.

Reply:
Makes sense to me. I actually didn't think the standard library `path.Join` was applicable, but upon checking it seems like it is.